composer.json

```diff
@@ -19,17 +19,17 @@
     ],
     "license": "MIT",
     "require": {
-        "illuminate/support": "^9.0",
-        "illuminate/container": "^9.0",
-        "illuminate/database": "^9.0",
-        "illuminate/events": "^9.0",
-        "mongodb/mongodb": "^1.11"
+        "illuminate/support": "^10.0",
```

Comment:
Would it make sense to support both Laravel 9.x and 10.x at the same time? I'm not too familiar with the amount of BC breaks that Laravel 10 will bring, but if it's possible to support both without a big hassle, users would profit from that.